orangecontrib/text/widgets/owdocumentembedding.py

```diff
@@ -149,8 +152,21 @@ def on_done(self, result):
 
     def on_exception(self, ex: Exception):
         self.cancel_button.setDisabled(True)
+        ex_msg = str(ex.__cause__ or ex).lower()  
+
         if isinstance(ex, EmbeddingConnectionError):
-            self.Error.no_connection()
```

Comment:
Looking at exception messages is ugly. But as mentioned in the comments of the other file, this will need to be changed anyway.